go-controller/pkg/util/net_linux_unit_test.go

```diff
@@ -0,0 +1,951 @@
+package util
+
+import (
+	"fmt"
+	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
+	netlink_mocks "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing/mocks/vishvananda/netlink"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/vishvananda/netlink"
+	"net"
+	"testing"
+)
```

Comment:
L4-12: plese group the modules -- first standard modules followed by the 3rd-party vendor modules.

Reply:
shall fix it